caffe2/python/convnet_benchmarks.py

```diff
@@ -80,25 +82,32 @@ def MLP(order, cudnn_ws):
                 dim_out=d,
                 weight_init=('XavierFill', {}),
                 bias_init=('XavierFill', {}),
+                WeightInitializer=initializer,
```

Comment:
Nice use of Initializer. But I think you can use `arg_scope` in `brew` module to make the whole code cleaner. PR #1093 is a good example of it.